docs/src/user_docs/assembly/io_operations.md

```diff
@@ -43,10 +43,10 @@ In both case the values must still encode valid field elements.
 
 | Instruction                         | Stack_input        | Stack_output        | Notes                                                                                                                                                                                                                                                                                                                          |
-| ----------------------------------- | ------------------ | ------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| ----------------------------------- | ------------------ | ------------------- |--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | adv*push.\_n* <br /> - _(n cycles)_ | [ ... ]            | [a, ... ]           | $a \leftarrow advstack.pop()$ <br /> Pops $n$ values from the advice stack and pushes them onto the operand stack. Valid for $n \in \{1, ..., 16\}$. <br /> Fails if the advice stack has fewer than $n$ values.                                                                                                               |
 | adv*loadw <br /> - *(1 cycle)\_     | [0, 0, 0, 0, ... ] | [A, ... ]           | $A \leftarrow advstack.pop(4)$ <br /> Pop the next word (4 elements) from the advice stack and overwrites the first word of the operand stack (4 elements) with them. <br /> Fails if the advice stack has fewer than $4$ values.                                                                                              |
-| adv*pipe <br /> - *(1 cycle)\_      | [C, B, A, a, ... ] | [E, D, A, a', ... ] | $[D, E] \leftarrow [adv\_stack.pop(4), adv\_stack.pop(4)]$ <br /> $a' \leftarrow a + 2$ <br /> Pops the next two words from the advice stack, overwrites the top of the operand stack with them and also writes these words into memory at address $a$ and $a + 1$.<br /> Fails if the advice stack has fewer than $8$ values. |
+| adv*pipe <br /> - *(1 cycle)\_      | [C, B, A, a, ... ] | [E, D, A, a', ... ] | $[D, E] \leftarrow [adv\_stack.pop(4), adv\_stack.pop(4)]$ <br /> $a' \leftarrow a + 8$ <br /> Pops the next two words from the advice stack, overwrites the top of the operand stack with them and also writes these words into memory at address $a$ and $a + 4$.<br /> Fails if the advice stack has fewer than $8$ values. |
```

Comment:
Not from this PR, but why do we write this instruction as `adv*pipe`? Shouldn't it be `adv_pipe`? And same question for `adv_push` and `adv_loadw`?